TaskManagement/project/views.py

```diff
@@ -19,12 +22,14 @@ class CreateProjectView(LoginRequiredMixin,CreateView):
 
 
     def get_form(self, *args, **kwargs):
-        try:
             form = super(CreateProjectView, self).get_form(*args, **kwargs)
-            form.fields['project_lead'].queryset = User.objects.filter(is_superuser=False)
-            return form
-        except Exception as error:
-            raise error
+            try:
+                form.fields['project_lead'].queryset = User.objects.filter(is_superuser=False)
+                return form
+            except Exception as error:
+                logging.error(str(e))
+                form.fields['project_lead'].queryset = User.objects.filter(is_superuser=False)
```

Comment:
does it make sense to write same line of code in except?
will it make any difference ?
won't it give same error?